src/panels/sidebar/vizOptions/index.jsx

```diff
@@ -27,6 +32,19 @@ const VizOptions = ({
   if (_.isBoolean(display) && !display) {
     return null;
   }
+
+  const updateVizOptionsWrapper = e => {
+    if (vizType === VIZ_TYPE_INTERACTIVEMARKER) {
+      updateVizOptions(key, {
+        topicName: e.target.value,
+        updateTopicName: undefined,
```

Comment:
Why are update and feedback topics set to undefined?
RViz only gives a selection for update topic. I guess it's reasonable if we do the same too.
The assumption doesn't seem to be too restrictive for selection through web interface.
The config itself can support a custom topic name but from a UX standpoint it seems better to not have a selection at all rather than having the user click thrice.

Reply:
Not sure how to get the topic names beforehand. I think its better to groupby type -> select index[0] if only 1 topic of that type exists.

Reply:
I am making an issue for this one. The `relatedTopics` thing is clunky and needs a PR to sort out this stuff.

Reply:
Resolved